Modules/CMakeLists.txt

```diff
@@ -5,17 +5,17 @@ add_subdirectory(Daq)
 add_subdirectory(Example)
 add_subdirectory(Benchmark)
 add_subdirectory(Skeleton)
-add_subdirectory(TOF)
-add_subdirectory(EMCAL)
+#add_subdirectory(TOF)
+#add_subdirectory(EMCAL)
 add_subdirectory(MUON)
-add_subdirectory(TPC)
-add_subdirectory(ITS)
-add_subdirectory(MFT)
-add_subdirectory(PHOS)
-add_subdirectory(FT0)
-add_subdirectory(TRD)
-add_subdirectory(HMPID)
-add_subdirectory(CPV)
-add_subdirectory(GLO)
-add_subdirectory(FV0)
-add_subdirectory(FDD)
+#add_subdirectory(TPC)
+#add_subdirectory(ITS)
+#add_subdirectory(MFT)
+#add_subdirectory(PHOS)
+#add_subdirectory(FT0)
+#add_subdirectory(TRD)
+#add_subdirectory(HMPID)
+#add_subdirectory(CPV)
+#add_subdirectory(GLO)
+#add_subdirectory(FV0)
+#add_subdirectory(FDD)
```

Comment:
Are you sure? :)

Reply:
I cannot speak freely :D
Thank you for spotting this obvious mistake.